deep_conmech/simulator/mesh/remesher.py

```diff
@@ -21,7 +21,7 @@ def bigger_or_zero(data):
     return data > -1e-05
 
 
-# @njit
+#@njit
```

Comment:
Remove this file from PR (not changed in fact)

Reply:
I changed it to only use #@njit - this way it is easier for me to find all functions that use numba

Reply:
Ok, but still is not related and it clutters PR